src/datamodel/character/character.rs

```diff
@@ -0,0 +1,24 @@
+pub trait Character {
+    fn get_team(&self) -> Team;
+    fn get_name(&self) -> CharactersName;
+    fn is_seen_by(&self, other: &CharactersName) -> bool;
+    fn can_see(&self, other: &CharactersName) -> bool;
+}
+
+#[derive(PartialEq, Debug)]
+pub enum CharactersName {
```

Comment:
Can we please rename this to `CharacterName` ?

Reply:
done.